projects/layer-selector/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "layer-selector",
+  "version": "0.0.1",
```

Comment:
let's do a `1.0.0-alpha.1` when it comes time to publish

Reply:
0ed0165